spec/models/comment_spec.rb

```diff
@@ -28,6 +28,12 @@
     it { is_expected.to belong_to(:comment_on) }
   end
 
+  describe "#with_associations" do
+    it "has associations" do
+      expect(Comment.with_associations).not_to be nil
```

Comment:
I don't know if this actually checks loading of associations. What do you think of:
```ruby
expect(Comment.all.first.association(:user).loaded?).to be false
expect(Comment.with_associations.first.association(:user).loaded?).to be true
```
This might detect association better and help detect when resources are not being `eager_loaded`